pkg/adapter/v2/cloudevents_test.go

```diff
@@ -253,10 +263,12 @@ func validateNotSent(t *testing.T, ce *test.TestCloudEventsClient) {
 	}
 }
 
-func validateMetric(t *testing.T, reporter source.StatsReporter, want int) {
+func validateMetric(t *testing.T, reporter source.StatsReporter, want int, eventType string) {
```

Comment:
Wonder if it would be cleaner to just add the wantRetryCount instead of baking in an assumption about the eventtype meaning something inside the test?